mlir/lib/Target/LLVMIR/DebugImporter.cpp

```diff
@@ -346,3 +385,21 @@ StringAttr DebugImporter::getStringAttrOrNull(llvm::MDString *stringNode) {
     return StringAttr();
   return StringAttr::get(context, stringNode->getString());
 }
+
+DistinctAttr DebugImporter::getOrCreateDistinctID(llvm::DINode *node) {
+  DistinctAttr &id = nodeToDistinctAttr[node];
+  if (!id)
+    id = DistinctAttr::create(UnitAttr::get(context));
+  return id;
+}
+
+llvm::function_ref<DIRecursiveTypeAttrInterface(DistinctAttr)>
```

Comment:
```suggestion
function_ref<DIRecursiveTypeAttrInterface(DistinctAttr)>
```
nit: I believe function_ref is available in the mlir namespace as well so the prefix may not be necessary?